{"url": "https://api.github.com/repos/datalad/datalad/issues/2720", "repository_url": "https://api.github.com/repos/datalad/datalad", "labels_url": "https://api.github.com/repos/datalad/datalad/issues/2720/labels{/name}", "comments_url": "https://api.github.com/repos/datalad/datalad/issues/2720/comments", "events_url": "https://api.github.com/repos/datalad/datalad/issues/2720/events", "html_url": "https://github.com/datalad/datalad/issues/2720", "id": 346315710, "node_id": "MDU6SXNzdWUzNDYzMTU3MTA=", "number": 2720, "title": "datalad extractors BROKEN >> [WARNING] Failed to load entrypoint neuroimaging: pybids 0.6.4 has no such extra feature 'analysis' [__init__.py:requires:2579]", "user": {"login": "tedstrauss", "id": 930560, "node_id": "MDQ6VXNlcjkzMDU2MA==", "avatar_url": "https://avatars.githubusercontent.com/u/930560?v=4", "gravatar_id": "", "url": "https://api.github.com/users/tedstrauss", "html_url": "https://github.com/tedstrauss", "followers_url": "https://api.github.com/users/tedstrauss/followers", "following_url": "https://api.github.com/users/tedstrauss/following{/other_user}", "gists_url": "https://api.github.com/users/tedstrauss/gists{/gist_id}", "starred_url": "https://api.github.com/users/tedstrauss/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/tedstrauss/subscriptions", "organizations_url": "https://api.github.com/users/tedstrauss/orgs", "repos_url": "https://api.github.com/users/tedstrauss/repos", "events_url": "https://api.github.com/users/tedstrauss/events{/privacy}", "received_events_url": "https://api.github.com/users/tedstrauss/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 10, "created_at": "2018-07-31T18:55:47Z", "updated_at": "2018-08-01T21:11:00Z", "closed_at": "2018-07-31T19:56:12Z", "author_association": "NONE", "active_lock_reason": null, "body": "#### What is the problem?\r\n\r\ndatalad aggregate-metadata with neuroimaging extractor types doesn't generate expcted metadata.\r\n\r\n`[WARNING] Failed to load entrypoint neuroimaging: pybids 0.6.4 has no such extra feature 'analysis' [__init__.py:requires:2579]`\r\n\r\n#### What steps will reproduce the problem?\r\n\r\n`datalad aggregate-metadata ...`\r\n\r\n#### What version of DataLad are you using (run `datalad --version`)? On what operating system (consider running `datalad wtf`)?\r\n\r\n```\r\n%> datalad wtf\r\n[WARNING] Failed to load entrypoint neuroimaging: pybids 0.6.4 has no such extra feature 'analysis' [__init__.py:requires:2579] \r\nDataLad\r\n=======\r\nVersion     : 0.10.1\r\nFull version: 0.10.1\r\n\r\nSystem\r\n======\r\nOS          : posix Linux 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018\r\nDistribution: Ubuntu/14.04/trusty\r\n\r\nLocale/Encoding\r\n===============\r\ndefault        : ascii\r\nfilesystem     : ISO-8859-1\r\nlocale.prefered: ISO-8859-1\r\n\r\nEnvironment\r\n===========\r\nPATH='/opt/sge/bin:/opt/sge/bin/lx-amd64:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/X11R6/bin:/usr/local/bin:/usr/local/mni/bin:/usr/local/matlab/bin:/opt/bin:/usr/local/bin:/usr/local/bic/bin:/usr/local/mni/bin'\r\nLANG='en_CA.ISO-8859-1'\r\nGIT_PYTHON_GIT_EXECUTABLE='/usr/lib/git-annex.linux/git'\r\nLC_ALL='en_CA.ISO-8859-1'\r\n\r\nExternals\r\n=========\r\nappdirs=1.3.0\r\nboto=2.20.1\r\ncmd:annex=6.20180329+gitga5fe62bed-1~ndall+1\r\ncmd:git=2.11.0\r\ncmd:system-git=1.9.1\r\ncmd:system-ssh=6.6.1p1\r\nexifread=2.1.2\r\ngit=2.1.8\r\ngitdb=2.0.0\r\nhumanize=0.5.1\r\niso8601=0.1.10\r\nmsgpack=0.5.6\r\nmutagen=1.41.0\r\nrequests=2.11.1\r\nsix=1.11.0\r\ntqdm=4.11.2\r\nwrapt=1.9.0\r\n\r\nInstalled extensions\r\n====================\r\nneuroimaging\r\n\r\nKnown metadata extractors\r\n=========================\r\nexif: OK\r\nannex: OK\r\ndicom: BROKEN (pybids 0.6.4 has no such extra feature 'analysis' [__init__.py:requires:2579])\r\nimage: OK\r\ndatalad_core: OK\r\nbids: BROKEN (pybids 0.6.4 has no such extra feature 'analysis' [__init__.py:requires:2579])\r\nnidm: BROKEN (pybids 0.6.4 has no such extra feature 'analysis' [__init__.py:requires:2579])\r\nnifti1: BROKEN (pybids 0.6.4 has no such extra feature 'analysis' [__init__.py:requires:2579])\r\nfrictionless_datapackage: OK\r\ndatalad_rfc822: OK\r\ndatacite: OK\r\naudio: OK\r\nxmp: OK\r\n\r\n```\r\n\r\n#### Is there anything else that would be useful to know in this context?\r\n#### Have you had any success using DataLad before? (to assess your expertise/prior luck.  We would welcome your testimonial additions to https://github.com/datalad/datalad/wiki/Testimonials as well)\r\n", "closed_by": {"login": "yarikoptic", "id": 39889, "node_id": "MDQ6VXNlcjM5ODg5", "avatar_url": "https://avatars.githubusercontent.com/u/39889?v=4", "gravatar_id": "", "url": "https://api.github.com/users/yarikoptic", "html_url": "https://github.com/yarikoptic", "followers_url": "https://api.github.com/users/yarikoptic/followers", "following_url": "https://api.github.com/users/yarikoptic/following{/other_user}", "gists_url": "https://api.github.com/users/yarikoptic/gists{/gist_id}", "starred_url": "https://api.github.com/users/yarikoptic/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/yarikoptic/subscriptions", "organizations_url": "https://api.github.com/users/yarikoptic/orgs", "repos_url": "https://api.github.com/users/yarikoptic/repos", "events_url": "https://api.github.com/users/yarikoptic/events{/privacy}", "received_events_url": "https://api.github.com/users/yarikoptic/received_events", "type": "User", "site_admin": false}, "reactions": {"url": "https://api.github.com/repos/datalad/datalad/issues/2720/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/datalad/datalad/issues/2720/timeline", "performed_via_github_app": null}